java-engine/src/main/java/io/getunleash/engine/WasmIsEnabledResponse.java

```diff
@@ -0,0 +1,7 @@
+package io.getunleash.engine;
+
+public class WasmIsEnabledResponse extends WasmResponse<Boolean> {
```

Comment:
I don't understand what this adds. From what I can tell it's just abstraction for abstraction's sake. Can we roll this back unless it adds something meaningful? The original generic response encodes what it needs in the type signature without hiding information